tests/sparql/eval/data-7.ttl

```diff
@@ -0,0 +1,18 @@
+PREFIX : <http://example/>
+
+# One each of each collection class.
```

Comment:
```suggestion
# One of each collection class.
```

Reply:
Doesn't read well but it is as meant.  It's not "one of the classes", its one instance of each class.

"One item of each of the collation classes." is clearer?

Reply:
My sentence does not say "one of the classes"; it effectively says "one [item] from each collection class."  (The word "item" is unnecessary, but can be included.)

"We have apples, oranges, and bananas. Which would you like?"
"One of each [collection class]."
Means you get one apple, one orange, and one banana ... which I believe is your intent.

Reply:
This is test data.
Changed to:
```
# The different kinds of orderable items.
# (Not in collation order).
```